src/components/EntityPage/File/index.js

```diff
@@ -323,12 +323,11 @@ const FileEntity = ({ api, fileId }) => {
       {file => {
         if (file.isLoading) {
           return <div>Loading</div>;
-        } else if (!file.data) {
-          return <div>no data</div>;
```

Comment:
Do we want something else for the no data information?

Reply:
Probably I'll check with Rosi